src/v/crypto/tests/crypto_bench.cc

```diff
@@ -70,32 +73,48 @@ struct openssl_perf {
     }
 };
 
-PERF_TEST_F(openssl_perf, md5_1k) {
+static std::unique_ptr<openssl_perf> global_perf{nullptr};
+
+struct openssl_perf_test {
+    openssl_perf_test() {
+        if (!global_perf) {
+            global_perf = std::make_unique<openssl_perf>();
+            ss::engine().at_exit([]() -> ss::future<> {
```

Comment:
caveat: I don't really have a good understanding of when/how these get run, whether a lambda coroutine here could be problematic at all, etc. But considering that we're operating on static data (and it's a benchmark 😛), probably fine?

Reply:
the more concerning this is why the lambda-coroutine wasn't caught by clang-tidy

Reply:
ah ha! nice thanks @rockwotj